ros/src/waypoint_updater/waypnt_updater.py

```diff
@@ -1000,6 +1004,8 @@ def set_waypoints_velocity(self):
                              # - could use to account for latency
         recalc = False       # indication that JMT calcs exceed bounds
 
+        boost_creep = 1.5  # creep speed is 1.5 times min_moving velocity
```

Comment:
`boost_creep` should probably be set as an instance variable `self.boost_creep` so the same variable can be referenced by set_waypoints_velocity() and set_creep() above.